orc8r/cloud/go/tools/swaggergen/generate/generate.go

```diff
@@ -165,6 +167,50 @@ func GenerateModels(targetFilepath string, configFilepath string, rootDir string
 	return nil
 }
 
+// GenerateStandAloneSpec generates a standalone spec file for a particular
+// Swagger spec.
+func GenerateStandAloneSpec(targetFilePath string, specs map[string]MagmaSwaggerSpec, outPath string) error {
+	absTargetFilepath, err := filepath.Abs(targetFilePath)
+	if err != nil {
+		return errors.Wrapf(err, "target filepath %s is invalid", targetFilePath)
+	}
+
+	var dependencies []string
```

Comment:
This is actually yamlSpecs, not just dependencies, because it includes the target spec